SPECS-EXTENDED/rubygem-hpricot/rubygem-hpricot.spec

```diff
@@ -0,0 +1,356 @@
+# Initially generated from hpricot-0.6.164.gem by gem2rpm -*- rpm-spec -*-
+%define	gem_name		hpricot
+
+# See bug 1402582
```

Comment:
Kind ask to clean up this spec a little, because these bug numbers are not from our system and will cause confusion in the future. Similar ask for any random comments, bug mentions, non-Mariner macros, etc. encountered below.

Reply:
fixed